lms/djangoapps/grades/management/commands/compute_grades.py

```diff
@@ -6,13 +6,14 @@
 
 import logging
 
-from django.core.management.base import BaseCommand
+from django.core.management.base import BaseCommand, CommandError
```

Comment:
CommandError is an unused import now, right?  I'm surprised pylint didn't complain.  Can you remove it?